doc/man3/OSSL_PARAM_get_int32_t.pod

```diff
@@ -0,0 +1,101 @@
+=pod
+
+=head1 NAME
+
+OSSL_PARAM_get_TYPE, OSSL_PARAM_set_TYPE, OSSL_PARAM_get_BN, OSSL_PARAM_set_BN
+- OSSL_PARAM helpers
```

Comment:
Shouldn't the OSSL_PARAM building macros be described as well?  Here or in another pod

Reply:
Yeah, they should.

Reply:
Minimal documentation added.
Added list termination macro `OSSL_PARAM_END`.